test/shared/src/main/scala/zio/test/package.scala

```diff
@@ -171,6 +171,12 @@ package object test extends CompileVariants {
   val assertCompletes: TestResult =
     assertImpl(true)(Assertion.isTrue)
 
+  /**
+   * Asserts that the given test was completed.
+   */
+  val assertCompletesM: UIO[TestResult] =
+    assertMImpl(UIO(true))(Assertion.isTrue)
```

Comment:
```suggestion
    assertMImpl(UIO.succeed(true))(Assertion.isTrue)
```
sorry to be pedantic, `UIO(x)` is when `x` is an effect.